### DISCUSSION TITLE
problem for chapter 2.4

### DISCUSSION THREAD

**Discussion by YoumiSum** [MathFoundationRL/Book-Mathematical-Foundation-of-Reinforcement-Learning#5] [General]
<img width="723" alt="无标题" src="https://github.com/MathFoundationRL/Book-Mathmatical-Foundation-of-Reinforcement-Learning/assets/50365816/9fd1aae1-5de6-46fd-883b-4b9e21ca93c9">

I have some problem in this...

**Comment by MathFoundationRL** [comment]
They are actually the same though the order of the sums is adjusted.

**Reply by Real-Mix** [reply]
why they are equal？I feel like I have trouble understanding the order of multiple summations... Maybe some details about related theorems should be included in the appendix (or I just miss that)

**Reply by MathFoundationRL** [reply]
This is a basic operation. To see it, the best way is to try some simple examples and verify.
